base/base/find_symbols.h

```diff
@@ -368,6 +384,12 @@ inline const char * find_first_symbols(std::string_view haystack, const SearchSy
     return detail::find_first_symbols_dispatch<true, detail::ReturnMode::End>(haystack, symbols);
 }
 
+template <char... symbols>
```

Comment:
Introducing "markdown" variants into such low-level code is quite ugly, conceptually speaking. Can't we call the existing `find_first_symbols()` function from WriteHelpers?